apps/data-types.md

```diff
@@ -662,6 +663,13 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | value | string | The key of an attribute, e.g. NeonPurple. |
 | displayValue | string | The localized displayed name of the attribute, e.g. Neon Purple. |
 
+## video
+
+| Attribute | Type | Description |
+| - | - |  - |
+| type | string | The Provider of a video. |
+| source | string | Specifies the video source. |
```

Comment:
add id in description

Reply:
| source | string | Specifies the video source. Can be the video id or video URL.